--- old
+++ new
@@ -17,0 +17,8 @@
+- [ ] stride egen modus for calibrering da dette må være et array bassert på hastighet 
+- [ ] fullscreen i dungeon mode
+- [ ] dungeon mode mangler overstyre hastighet, trengs dette? 
+- [ ] trang med plass i høyde på dungeon boss precision
+- [ ] boss endurance 133+ på easy blir for stor overgang 
+- [ ] link clients trenger litt css
+- [ ] fallback / static streetview mangler controls / har feil hud 
+- [ ] Youtube lobby selcted video flytter alt ned out of recht så kan ikke endre
@@ -21,1 +29,7 @@
-- [x] route mangler duration 
+- [x] route mangler duration
+- [ ] Youtube starter ikke om pause for lenge. Trenger en manuell knapp for play
+- [ ] Youtube hud. Ingen space mellom labels
+- [ ] route er blitt en rett linje igjen...
+- [ ] route har feil hud og mangler controls
+- [ ] når client forlater avsluttetes ikke realm
+- [ ] 